### PR TITLE
Chore: Update Regionalization Input

### PR DESCRIPTION
**Pull request by saranicoly**
## What's the purpose of this pull request?

It updates the Regionalization Input to use `InputText` component.

Before:

image

After:

image
image

## How to test it?

**Option 1:** Opening the Regionalization Modal
**Option 2:** Calling the Regionalization Input wherever you want to.

## References

- [Input Text's PR](https://github.com/vtex-sites/base.store/pull/440)

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] CHANGELOG entry added

### CODE REVIEW

File: src/components/ui/InputText/InputText.tsx [hunk 1]
```diff
@@ -52,13 +53,13 @@ const InputText = ({
   buttonActionText = 'Apply',
   onSubmit,
   placeholder = ' ', // initializes with an empty space to style float label using `placeholder-shown`
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  inputRef = useRef<HTMLInputElement>(null),
```
Comment (saranicoly) [inline src/components/ui/InputText/InputText.tsx:57]: I'm accepting suggestions of other approaches for that.

File: src/components/ui/InputText/InputText.tsx [hunk 2]
```diff
@@ -18,6 +18,10 @@ export type InputTextProps = {
    * The error message is displayed when an error occurs.
    */
   errorMessage?: string
+  /**
+   * Component's ref.
+   */
+  inputRef?: React.RefObject<HTMLInputElement>
```
Comment (lucasfp13) [inline src/components/ui/InputText/InputText.tsx:24]:
```suggestion
  inputRef?: MutableRefObject<HTMLInputElement | null>
```

File: src/components/ui/InputText/InputText.tsx [hunk 3]
```diff
@@ -52,13 +56,13 @@ const InputText = ({
   buttonActionText = 'Apply',
   onSubmit,
   placeholder = ' ', // initializes with an empty space to style float label using `placeholder-shown`
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  inputRef = useRef<HTMLInputElement>(null),
```
Comment (lucasfp13) [inline src/components/ui/InputText/InputText.tsx:60]:
```suggestion
  inputRef,
```

Comment (lucasfp13) [reply src/components/ui/InputText/InputText.tsx:60]: and at line 72:

``
inputRef?.current?.focus()
``

**Review by lucasfp13 — COMMENTED** [review]
LGTM!

I left the suggestion over the `useRef` approach, what do you think?

**Comment by filipewl** [comment]
Closed in favor of https://github.com/vtex-sites/gatsby.store/pull/8 and https://github.com/vtex-sites/gatsby.store/pull/9.

**Pull request closed without merging**